components/table/stories/table.stories.js

```diff
@@ -394,17 +524,44 @@ Scrollable.parameters = {
  * A table can also be made up of `div` tags if needed, instead of a `<table>`. This example uses both the div markup, and the scrollable wrapper.
  */
 export const DivsScrollable = Template.bind({});
-DivsScrollable.storyName = "Scrollable with divs";
+DivsScrollable.storyName = "Scrollable table with divs";
 DivsScrollable.args = {
 	...Scrollable.args,
 	useDivs: true,
-	rowItems: ExampleRowItems,
 };
 DivsScrollable.tags = ["!dev"];
 DivsScrollable.parameters = {
 	chromatic: { disableSnapshot: true },
 };
 
+
+/**
+ * Thumbnails, avatars, and other icons can be used in the table content, with some additional markup and classes for alignment.
+ */
+export const Visuals = Template.bind({});
+Visuals.args = {
+	rowItems: [
+		{
+			cellContent: "Avatar example",
+			visualElement: "avatar",
+		},
+		{
+			cellContent: "Icon example",
+			visualElement: "icon",
+		},
+		{
+			cellContent: "Thumbnail example",
+			visualElement: "thumbnail",
+		},
+	],
+};
+Visuals.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+Visuals.storyName = "With visuals";
+
+
+// TODO: The design team doesn't have support for collapsible rows in the table component, so they are removed from the docs page for now.
```

Comment:
I have a couple `TODO` notes throughout the stories file here, reflecting on the conversation I had with Stefan. 

Design doesn't have collapsible row designs, or dropzone designs done (and they're unsure if they will). React only has beta drag & drop table components. 

So I didn't really want to get rid of our stories or tests for those table variants/options. What are our thoughts on just removing these 5 or 6 stories regarding collapsible rows and dropzones from the autodocs?